user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Installing_DataMiner_Cube/DataMiner_Cube_deployment_methods.md

```diff
@@ -133,6 +133,8 @@ Below you can find an overview of the different methods that can be used to depl
 
 - The launcher is backwards compatible and can be used to deploy any 9.x or 10.x Cube version. Shared MSI installation packages can be provided on demand for older versions.
 
+- From DataMiner 10.4.0 [CU16]/10.5.0 [CU4]/10.5.7 onwards<!-- RN 42706 -->, the *About* window of the start window (available via the cogwheel button) will show more info about how the installed version was deployed. If it shows "(shared)", a shared MSI installation was used. If it shows ("MSI"), a bootstrap MSI installation was used.
```

Comment:
My apoligies, the new suffixes are "shared" and "Bootstrap", so no longer "MSI".
I forgot to update the release note, my bad!
FYI Why this is changed: the information must reflect and be aligned with following docs article:
https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Installing_DataMiner_Cube/DataMiner_Cube_deployment_methods.html